impl/pom.xml

```diff
@@ -27,17 +27,23 @@
    <modelVersion>4.0.0</modelVersion>
 
    <parent>
-        <groupId>org.glassfish</groupId>
+        <groupId>org.wildfly.faces</groupId>
         <artifactId>mojarra-parent</artifactId>
-        <version>4.0.1-SNAPSHOT</version>
-    </parent>
+        <version>4.0.0.SP01-SNAPSHOT</version>
```

Comment:
Just to check, this was based off 4.0.0-M6, right? I'm guessing there was just an extra commit in their branch that bumped to 4.0.1?